runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesConfig.java

```diff
@@ -34,12 +34,25 @@
 public interface InterfaceStylesConfig extends Config
 {
 	@ConfigItem(
-		keyName = "gameframe",
-		name = "Gameframe",
-		description = "The gameframe to use for the interface"
+			position = 1,
+			keyName = "gameframe",
+			name = "Gameframe",
+			description = "The gameframe to use for the interface"
 	)
 	default Skin skin()
 	{
-		return Skin.AROUND_2010;
+		return Skin.DEFAULT;
```

Comment:
This should probably not be changed. Since the plugin is still disabled by default, enabling it should have an immediate change instead of requiring further configuration to see any changes.

Reply:
No longer uses InterfaceStyles plugin. Reverted all changes